addons/sms/tests/common.py

```diff
@@ -28,7 +28,7 @@ def _contact_iap(local_endpoint, params):
                 } for number in params['numbers']]
                 return True  # send_message v0 API returns always True
             # mock batch sending
-            if local_endpoint == '/iap/sms/1/send':
+            if local_endpoint == '/iap/sms/2/send':
```

Comment:
Actually I think the mock could support both of them, isn't it ? If some external people want to use it (I doubt it, but hey). Is the 'unregistered' new error the only difference from addons point of view ?

Reply:
As explained in my other message, yes it's the only difference :) I guess we ould support both in this case